### ISSUE TITLE
DateTimePicker layout problem in IE 11

### ISSUE THREAD

**Issue by robert-lindstrom** [SharePoint/sp-dev-fx-controls-react#301]
#### Category
[ ] Enhancement

[X] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [latest]

#### Expected / Desired Behavior / Question
The time row of the DateTimePicker should stay within its area
![DateTimePickerLayoutOk](https://user-images.githubusercontent.com/45068174/57786956-c2b29900-7734-11e9-89b3-8e80a3aa59c4.png)

#### Observed Behavior
The time row of the DateTimePicker overflows in IE 11 (se picture) due to issues with flexbox layout.
![DateTimePickerLayoutIssue](https://user-images.githubusercontent.com/45068174/57786472-d4477100-7733-11e9-92eb-e0f21e3d365a.png)

#### Steps to Reproduce
Put two DateTimePickers next to each other in a grid and check layout using IE 11:
```jsx
<div className="ms-Grid-row">
  <div className="ms-Grid-col ms-lg6">
    <DateTimePicker label="DateTime Picker without seconds" showSeconds={false} onChange={(value) => console.log("DateTimePicker value:", value)} />
  </div>
  <div className="ms-Grid-col ms-lg6">
    <DateTimePicker label="DateTime Picker with seconds" showSeconds={true} onChange={(value) => console.log("DateTimePicker value:", value)} />
  </div>
</div>
```
Thanks!

**Comment by estruyf** [comment]
Thanks @robert-lindstrom for reporting and fixing this.

**Comment by estruyf** [comment]
This has just been released in version `1.13.2`
